### DISCUSSION TITLE
Get Node Description and Datatype

### DISCUSSION THREAD

**Discussion by bibhudatta-sahu** [eclipse-milo/milo#996] [General]
Hello Everyone,
         I would like to know how to get description and datatype of a node while browsing address space. Could you please help me on it.

Thanks,
Bibhudatta

**Comment by kevinherron** [comment]
This information isn’t available using the Browse service alone.

Typically a client would look at the NodeClass of the Nodes it browsed and then use the Read service to get attributes like Description, DataType, ValueRank, and ArrayDimensions for Variable Nodes.

**Reply by bibhudatta-sahu** [reply]
Thank you for the solution. Could you please guide by providing small sample code to refer.

**Reply by bibhudatta-sahu** [reply]
Thanks @kevinherron   I got the solution after analyzing the MILO sdk code. 👍
